docs/source/distributed_simulation.rst

```diff
@@ -87,27 +89,27 @@ Basic flows between client and server
 | **(b-2)** `URRROS2SimulationstateClient <doxygen_generated/html/d7/d6a/class_u_r_r_r_o_s2_simulation_state_client.html>`_ in the client pass the request to the server via RPC
 | **(b-3)** SimulationStateClient in the server pass the request to the `ASimulationState <doxygen_generated/html/d2/dde/class_a_simulation_state.html>`_
 | **(b-4)** SimulationState spawn the robot in the server.
-| **(b-5)** Spawned robot is replicated to the clients. Only In the client which send spawn request, robot create/initiate ROS2 compoenent. No ROS components are created/initiated in other clients.
+| **(b-5)** Spawned robot is replicated to the clients. Only in the client which sends spawn request, robot creates/initiates ROS2 compoenent. No ROS components are created/initiated in other clients.
 
 
 | **Robot Movement flow**
 | **(c-1)** ROS2 application publish command topic such as /cmd_vel
 | **(c-2)** Robot in the client moves to follow command.
-| **(c-3)** Set server robot movement is requested to  `ARRNetworkPlayerController <doxygen_generated/html/db/d54/class_a_r_r_network_player_controller.html>`_ 
+| **(c-3)** Set server robot movement is requested by `ARRNetworkPlayerController <doxygen_generated/html/db/d54/class_a_r_r_network_player_controller.html>`_ 
```

Comment:
Robot requests to set server robot movment to ARRNetworkPlayerController

maybe better? I don't have confidence in English but point is Robot send requests to NetworkPlayercontroller

Reply:
Okay, this is very clear from Figure 2. You're right then I would propose "(c-3) Robot requests ARRNetworkPlayerController to set server robot movement".